generator.yml

```diff
@@ -303,6 +303,7 @@ configs:
   - &riscv_no_efi      {config: [defconfig, CONFIG_EFI=n],                                                     kernel_image: Image,        ARCH: *riscv-arch,      << : *kernel}
   - &riscv_alpine      {config: *riscv-alpine-config-url,                                                      kernel_image: Image,        ARCH: *riscv-arch,      << : *kernel}
   - &riscv_suse        {config: *riscv-suse-config-url,                                                        kernel_image: Image,        ARCH: *riscv-arch,      << : *kernel}
+  - &riscv_gki         {config: gki_defconfig,                                                                 kernel_image: Image,        ARCH: *riscv-arch,      << : *kernel}
```

Comment:
Just building and booting `gki_defconfig` works for me locally; I guess the additional configuration fragments are not strictly necessary for the simple boot testing that we do with `boot-qemu.py` (perhaps they are for booting cuttlefish?).

If we want to be a little closer to ACK, this could be

```
config: [gki_defconfig, 64-bit.config, gki.config],
```

which should do the right thing (it does for `tuxmake`, I am not sure if `tuxsuite` has additional checks or restrictions). Up to you.

Reply:
you can combine configs like that? I guess today we only use explicit `CONFIG_XXX=y` syntax for individual configs.

Oh right, that's what we do for CrOS configs (`arm64-cros-configs`, `x86_64-cros-configs`). Good idea.